scripts/subpath-workaround.mjs

```diff
@@ -30,6 +30,11 @@ async function run() {
     ...subpathHelperFile.ignoredFolders,
     'dist',
   ];
+
+  if (pkgFile.files.length !== allFilesNames.length) {
+    throw new Error('The package.json "files" array length does not match the subpaths.mjs');
+  }
+
```

Comment:
❓ is this almost the same error as the one that exists right below?

Reply:
The `.every` error won't fire if the length is not the same, so early return here and make the error clearer

Reply:
We could merge these 2 checks and throw a single error if any of these checks fail.